fractal_server/app/routes/api/v2/_aux_functions_history.py

```diff
@@ -83,10 +85,27 @@ def read_log_file(
     try:
         with open(logfile) as f:
             return f.read()
-    except Exception as e:
+    except OSError as e1:
```

Comment:
We can use `if Path(logfile).exists() / else` directly, so that we don't end up capturing a broader `OSError` class.